ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TutorialLiveShareLayout.kt

```diff
@@ -0,0 +1,157 @@
+            .fillMaxSize()
+            .padding(
+                top = dimensionResource(R.dimen.tutorial_page_inset_top),
+                bottom = dimensionResource(R.dimen.tutorial_page_inset_bottom)
+            ).verticalScroll(rememberScrollState())
+    ) {
+        Spacer(modifier = Modifier.weight(1f))
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Column() {
+                Spacer(modifier = Modifier.minLinesHeight(1, MaterialTheme.typography.titleLarge))
+                Spacer(modifier = Modifier.minLinesHeight(5, MaterialTheme.typography.bodyMedium).padding(top = 16.dp))
+            }
+
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+
+                Text(
+                    text = stringResource(title),
+                    style = MaterialTheme.typography.titleLarge,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp),
+                    color = MaterialTheme.colorScheme.primary
+                )
+
+                Text(
+                    stringResource(body),
+                    style = MaterialTheme.typography.bodyMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp).padding(top = 16.dp)
+                )
+            }
+        }
+        /*
+        Spacer for first two elements only
+        Animation will not be in box, will be element of parent column
+
+         */
+
+        when {
+            anim != null -> {
+                val composition by rememberLottieComposition(
+
+                    LottieCompositionSpec.RawRes(anim)
+                )
+
+                val progress by animateLottieCompositionAsState(
+                    // pass the composition created above
+                    composition,
+
+                    // Iterates Forever
+                    iterations = LottieConstants.IterateForever,
+
+                    // pass isPlaying we created above,
+                    // changing isPlaying will recompose
+                    // Lottie and pause/play
+                    isPlaying = true,
+
+                    // pass speed we created above,
+                    // changing speed will increase Lottie
+                    speed = 1f,
+
+                    // this makes animation to restart
+                    // when paused and play
+                    // pass false to continue the animation
+                    // at which is was paused
+                    restartOnPlay = false
+
+                )
+                LottieAnimation(
+                    composition,
+                    { progress },
+                    modifier = Modifier
+                        .height(dimensionResource(R.dimen.tutorial_page_live_share_anim_height))
+                        .fillMaxWidth().padding(top = 16.dp)
```

Comment:
padding needs to be defined before you set the height of the animation. See my previous comment about how modifier order matters.